crt_portal/cts_forms/templates/forms/widgets/crt_date_entry.html

```diff
@@ -0,0 +1,14 @@
+<div class="input-group date" id="{{ widget.datepicker_id }}" data-target-input="nearest">
+  {% include "django/forms/widgets/input.html" %}
+  <div class="input-group-append" data-target="#{{ widget.datepicker_id }}" data-toggle="datepicker">
+    <div class="input-group-text"><i class="fa fa-calendar"></i></div>
+  </div>
+</div>
+
+<script nonce="{{ request.csp_nonce }}">
+  $(function () {
```

Comment:
We do not have access to jQuery here -- assuming this is what you intended.

So we get an `ReferenceError: $ is not defined` in the console. Can you rewrite this without using jQuery? As is, this would break in IE11, I believe (but not tested).

Reply:
This doesn't work in ie11, yes, just checked. Need to re-do this with just html5 tags.

Reply:
Reverted to using Django date fields with input type="date". Works as expected.

Reply:
this file is removed

Reply:
@Jkrzy @cantsin, the django dateinput (with <input type=date>) widget doesn't render calendar icon or the calendar on ie11, works fine on chrome, firefox, edge and safari. This is not using any javascript or jquery. This is a blocker right now if ie11 is a display is a must, and am looking for help. any suggestions are greatly appreciated. adding @hakhalid11 and @MattBowles8.

Reply:
IE 11 and safari is unable to display input_type = 'date' fields, they display as text. We can show a hint of what to enter, but in case of garbage or ill-formatted data, we can ignore, and warn users with an alert or message box. I think this is the best option considering  not using javascript/jquery widgets and we want to support ie11/safari. The other option is to break down date field in to separate day, month and year field. Let's talk at noon @Jkrzy , @cantsin , @hakhalid11 and @MattBowles8